MAINTAINERS.md

```diff
@@ -0,0 +1,199 @@
+- **Provides Context:** If background knowledge is required, this should be explicitly mentioned and a list of suggested readings included.
+
+- **Gives Examples:** If possible, link to examples of similar implementations so new contributors have a reference guide for their changes.
+
+- **Identifies Relevant Code:** The relevant code and tests to be changed should be linked in the issue.
+
+- **Ready to Test:** There should be existing tests that can be modified, or existing test cases fit to be copied. If the area of code doesn't have tests, before labeling the issue, add a test fixture. This prep often makes a great help wanted task!
+
+Maintainers are encouraged to invest the extra effort to write and shepherd along these issues and PRs, since onboarding reliable, long-term contributors is an excellent foundation for a sustainable, long-running project that doesn’t burn out the participants.
+
+**Measures can include:**
+- Help with the contributor’s dev setup, and if you realise that the onboarding documentation for developers is lacking or out of date, assign their improvement to the new contributor! They probably took notes on the process anyway.
+- Proactively watch open first timer-PRs as they change and get in touch if you can save people some time or pain.
+- Invite them to meetings and chats
+- Rope in other maintainers for a second LGTM at the end
+- If the contributor agrees, publicly thank them on whichever channels you broadcast on, as well as in the release notes
+- Suggest related `Help wanted` issues
+
+Unless the contributors are insecure about public communication, you’re encouraged to _not_ use private messages as much as possible. Keeping communication public ensures that other people can find and benefit from your discussions in the future.
+
+#### Other Issue Types
+
+There are a variety of additonal labels that are generally self-explanatory, for example:
+
+- `Feature`
+- `Documentation`
+- `Performance`
+- `Testing`
+- `Cleanup`
+- `Security`
+- `Compatibility`
+
+A full list can always be found on the project’s [labels page](https://github.com/openpgpjs/openpgpjs/labels).
+
+### Step 2: Prioritize the Issue
+
+Aside from a type label and the housekeeping labels, issues that are or should become actionable also benefit from having a priority label.
+
+#### `priority/critical-urgent`
+
+Something has gone 💥. Stuff is on fire. Drop what you’re doing and work on this next. Must be in the next planned release, or requires its own release.
+
+#### `priority/important-soon`
+
+Must be staffed and worked on either currently or very soon—ideally in time for the next release. Important, but wouldn't block a release.
+
+#### `priority/important-longterm`
+
+Important over the long term, but may not be currently staffed and/or may require multiple releases to complete. Wouldn't block a release.
+
+#### `priority/backlog`
+
+Would be nice, but no one has the resources to work on it. Good candidates for the `Help wanted` label.
+
```

Comment:
We don't really prioritize issues in OpenPGP.js; because the priority of issues is specific to each application: what's important to Proton might not be important to someone else, and vice versa. So we don't really want to categorize whether any given issue is "objectively" important or not.

Also, issues are not staffed by OpenPGP.js directly, but rather applications and contributors decide what they work on.

```suggestion
```

Reply:
I think I understand what you mean to say, but since this is a public forum, I'd like to clarify that we do security and priority assessment of issues, and immediately tackle those with high priority.
In this sense, "critical-urgent" and "important-soon" will typically be fixed by us [maintainers] in a matter of days, without the need to add labels (esp. if there is some confidentiality involved in the case of security issues).

I feel your reply is primarily applicable to "feature requests", rather than bugs/issues in the proper sense 🙂